runtime/commonMain/src/kotlinx/serialization/Mapper.kt

```diff
@@ -1,173 +0,0 @@
-/*
- * Copyright 2017-2020 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
- */
-
-package kotlinx.serialization
-
-import kotlinx.serialization.CompositeDecoder.Companion.READ_DONE
-import kotlinx.serialization.modules.*
-
-class Mapper(context: SerialModule = EmptyModule) : AbstractSerialFormat(context) {
```

Comment:
Please leave this (empty) class here with a deprecation that suggests adding the new module

Reply:
Why we haven't done the same for Protobuf and CBOR?

Reply:
We did it for ProtoBuf via `SerialId`.
We can't do it for `ProtoBuf` and `Cbor` classes because they weren't renamed, thus they will always conflict with each other (a deprecated declaration with a new one)